### PR TITLE
Maven runner implementation

### PR DESCRIPTION
**Pull request by Sy1v4in**
This runner builds the project using the `mvn install` command and run the test using the `mvn test` command.
The run is a bit slow due to maven time to start.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 echo 'CG> message Compile and run your code'
```
Comment (CGNonofr) [inline entrypoint.sh]: why not just `echo 'message Compile and run your code'`?

File: entrypoint.sh [hunk 2]
```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
+echo 'CG> message Compile and run your code'
```
Comment (CGNonofr) [inline entrypoint.sh]: why not just `echo 'message Compile and run your code'`?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+First, this runner compiles the project thanks to maven.
+
+At each play, it runs a specific test using the `mvn test -Dtest=XXX` command.
+
+
+# How to Use
+
+To use this runner for your project, edit the `codingame.yml` file and add the following lines to your project:
+
+    runner:
```
Comment (BeLi4L) [inline README.md]: You could add YAML syntax highlighting :P

File: entrypoint.sh (hunk 2 above)
Comment (Sy1v4in) [reply entrypoint.sh]: Why 'message' in this case?

File: README.md (hunk 3 above)
Comment (Sy1v4in) [reply README.md]: Fix in f65984d6f60a0ca9684629aa9ff8e573537a9c3b

File: entrypoint.sh (hunk 2 above)
Comment (Sy1v4in) [reply entrypoint.sh]: Right, why? ;-) 766d79fb56d72710704ecae10d28eee49c822fed

**Review by CGNonofr — APPROVED** [review]
LGTM

**Pull request merged**